lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -322,3 +322,11 @@ section.foldit {
 
    }
 }
+
+#courseware-results-wrapper {
```

Comment:
NIT: if possible, it would be great to use class based styles instead of an ID selector here.